src/Microsoft.Azure.WebJobs.Host/Common/GenericBinder.cs

```diff
@@ -10,17 +10,39 @@
 using Microsoft.Azure.WebJobs.Host.Protocols;
 using Microsoft.Azure.WebJobs.Host.Triggers;
 
-namespace Microsoft.Azure.WebJobs.ServiceBus
+namespace Microsoft.Azure.WebJobs.Host
 {
-    // Helpers for doing general-purpose bindings. 
-    internal class GenericBinder
+    /// <summary>
+    /// Helpers for creating bindings to common patterns such as Messaging or Streams.
+    /// </summary>
+    public static class GenericBinder
     {
+        /// <summary>
+        /// Bind a  parameter to an IAsyncCollector. Use this for things that have discrete output items (like sending messages or writing table rows)
+        /// This handles the various flavors of parameter types and will morph them to the connector.
```

Comment:
I'll update the comments. It means that it will add additional adapters to connect the user's parameter type  to an IAsyncCollector. For example, if the user is a ICollector<T>, this will add an adapter that implements ICollector<T> and calls IAsyncCollector<T>. 

---
In reply to: [54931918](https://github.com/Azure/azure-webjobs-sdk/pull/643#discussion_r54931918) [](ancestors = 54931918)